demo/angular/src/app/voice_widget.component.ts

```diff
@@ -106,4 +80,28 @@ export class VoiceWidget {
     this.inference = null
     this.rhinoService.pushToTalk()
   }
+
+  public async initEngine(accessKey: string) {
+    if (accessKey.length === 56) {
```

Comment:
this is tricky. Why do we need this?

Reply:
Makes the demo a little nicer, otherwise as users type in their `AccessKey` they'll get an `invalid key error` pop-up

Reply:
I've tweaked it to check if length is greater than 0, now it aligns with the other demos.